internal/ansible/watches/watches_test.go

```diff
@@ -892,3 +892,49 @@ func TestGetPossibleRolePaths(t *testing.T) {
 		})
 	}
 }
+
+func TestReplaceEnvVars(t *testing.T) { //nolint:gocyclo
```

Comment:
Could you elaborate on the reasoning for the nolint here?

Reply:
Copy-paste :) I copied the TestLoad function when I started writing my test function, and [it has that](https://github.com/operator-framework/ansible-operator-plugins/blob/main/internal/ansible/watches/watches_test.go#L108C1-L109C1). No reason other than that. I can delete that.

Reply:
this is done in latest commit